docs/docs/manual/columnediting.md

```diff
@@ -0,0 +1,128 @@
+
+**"HTTP error 404 : Not Found"** indicates that the information you are requesting does not exist, perhaps due to a problem with your cell values if it only happening in certain rows.** "HTTP error 500 : Internal Server Error"** indicates the remote server is having a problem filling your request. You may wish to simply wait and try again later, or double-check the URLs. 
+
+## Add columns from reconciled values
+
+The “Add columns from reconciled values…” feature operates on reconciled columns in OpenRefine - that is, columns containing data that has been through [reconciliation](#reconcile) processes. See that section for more details about how to get your data into a reconciled format. 
+
+Once you have pulled reconciliation values and selected one for each cell, selecting “Add column from reconciled values” will bring up a window to choose which information you’d like to generate into a new column. If you have reconciled against Wikidata, for example, you will see the option of adding their unique identifiers as a new column (automatically named “Qid”). 
+
+The quality of the suggested properties will depend on how you have reconciled your data beforehand: reconciling against a specific type will provide you with suggested properties of that type.
+
+![A screenshot of the settings window for adding reconciled values.](/img/columnreconciled.png)
+
+If you have left any values unreconciled in your column, you will see “&lt;not reconciled>” in the preview. These will generate blank cells if you continue with the column addition process. 
+
+One reason to reconcile a dataset to some database is that it allows you to pull data from the database into OpenRefine. There are two ways to do this:
+
+*   If the reconciliation service supports the[ Data Extension API](https://github.com/OpenRefine/OpenRefine/wiki/Data%20Extension%20API), then you can use the "Add columns from reconciled values" action to augment your OpenRefine project with new columns. For example, if you have a column of chemical elements identified by name, you can fetch categorical information about them such as their atomic number and their element symbol, as the animation shows below.
+
+![A screenshare of elements fetching related information.](/img/reconcileelements.gif)
+
+*   If the reconciliation service does not support this feature, look out for a generic web API for that data source. You can use the "Add column by fetching URLs" operation to call this API with the IDs obtained from the reconciliation process. For instance, if you have used the OpenCorporates service, you can use[ their API](https://api.opencorporates.com/documentation/API-Reference) to fetch JSON-formatted data about the reconciled companies. The GREL expression `"https://api.opencorporates.com/companies/"+cell.recon.match.id` will be translated to URLs that can then be parsed in OpenRefine.
```

Comment:
It might be better to use a non-commercial example like VIAF here (plus, we probably have more librarian users)

Reply:
I started with VIAF but I didn't love their metadata formats as a concrete example. I went with the Getty ULAN instead. I would love some feedback on this choice though:

https://deploy-preview-3103--openrefine-docs.netlify.app/manual/reconciling#add-columns-by-fetching-urls

> If the reconciliation service cannot extend data, look for a generic web API for that data source, or a structured URL that points to their dataset entities via unique IDs (such as https://viaf.org/viaf/000000). You can use the "Edit column" → "Add column by fetching URLs" operation to call this API or URL with the IDs obtained from the reconciliation process. This will require using GREL expressions.
> 
> You will likely not want to pull the entire HTML content of the pages at the ends of these URLs, so look to see whether the service offers a metadata endpoint, such as JSON-formatted data. You can either use a column of IDs, or you can pull the ID from each matched cell during the fetching process.
> 
> For example, if you have reconciled artists to the Getty's ULAN, and have their unique ULAN IDs as a column, you can generate a new column of JSON-formatted data by using "Add column by fetching URLs" and entering the GREL expression `"http://vocab.getty.edu/" + value + ".json"` in the window. For this service, the unique IDs are formatted "ulan/000000" and so the generated URLs look like "http://vocab.getty.edu/ulan/000000.json."
> 
> You can use the Add entity identifiers column operation above if you don't already have a column of unique IDs.
> 
> You can alternatively insert the ID directly from the matched column using a GREL expression like `"http://vocab.getty.edu/" + cell.recon.match.id + ".json"` instead.
> 
> Remember to set an appropriate throttle and to refer to the service documentation to ensure your compliance with their terms.